app/src/main/java/org/cru/godtools/fragment/ToolDetailsFragment.java

```diff
@@ -280,6 +289,20 @@ private void updateViews() {
             mTitle.setText(ModelUtils.getTranslationName(getContext(), mLatestTranslation, mTool));
         }
         bindShares(mShares, mTool);
+        if (mBanner != null) {
+            mBanner.asImageView().setVisibility(
```

Comment:
Since this is being added, how was the banner being set to visible before this?

Reply:
@wrandall22 The banner was always visible before. Now that we are displaying it only when there is no video present I wanted to remove it from the view.